### PR TITLE
CBG-427: SGCollect_info https try

### PR DESCRIPTION
**Pull request by JRascagneres**

### CODE REVIEW

File: tools/sgcollect_info [hunk 1]
```diff
@@ -571,11 +577,35 @@ def main():
...
+    else:
+        if not sg_url:
+            root_url = "127.0.0.1:4985"
+        else:
+            root_url = sg_url
+        sg_url_tmp = "http://" + root_url
+        print("Trying Sync Gateway URL: {0}".format(sg_url_tmp))
+
+        try:
+            requests.get(sg_url_tmp)
```
Comment (bbrks) [inline tools/sgcollect_info]: We're importing a new library in order to do this request. All others use `urllib2.urlopen`. Are we able to use that instead?

File: tools/sgcollect_info [hunk 2]
```diff
@@ -571,11 +577,35 @@ def main():
...
+        except requests.ConnectionError, e:
+            print("Failed to communicate with: {} {}".format(sg_url_tmp, e))
+            sg_url_tmp = "https://" + root_url
+            print("Trying Sync Gateway URL: {0}".format(sg_url_tmp))
+            try:
+                requests.get(sg_url_tmp)
+            except requests.ConnectionError, e:
+                print("Failed to communicate with: {} {}".format(sg_url_tmp, e))
```
Comment (bbrks) [inline tools/sgcollect_info]: copy/paste leftover?

**Review by bbrks — CHANGES_REQUESTED** [review]

File: tools/sgcollect_info (hunk 1 above)
Comment (JRascagneres) [reply tools/sgcollect_info]: No. The urlopen didn't try to communicate with the endpoint so much as it checked if it was alive. Ie. It'd say all is good but when you try to use http but you'd get a handshake error in SG. Whereas requests actually tries to get data from the point.

Comment (bbrks) [reply tools/sgcollect_info]: Are you sure? What about this use from further up (or everywhere else we get data out of SG over the REST API)?

The docs seem to suggest it'll perform a HTTP GET request, unless explicitly set to POST https://docs.python.org/2/library/urllib.html#urllib.urlopen 

```python
response = urllib2.urlopen(expvar_url(sg_url))
data = json.load(response)
```

File: tools/sgcollect_info [hunk 3]
```diff
@@ -15,6 +15,8 @@ import uuid
 
 from sys import platform as _platform
 
+from botocore.vendored import requests
```
Comment (bbrks) [inline tools/sgcollect_info]: Update to remove new import based on https://github.com/couchbase/sync_gateway/pull/4178#discussion_r298133190

**Review by bbrks — CHANGES_REQUESTED** [review]

**Review by bbrks — APPROVED** [review]
LGTM 👍

**Pull request merged**